### ISSUE TITLE
[Request] Stream.me Implementation

### ISSUE THREAD

**Issue by Draygera** [xedoc/Ubiquitous2#15]
On Stream.me, there is an optional Anonymous viewer chat flag for streamers that want to let non-registered viewers to chat. Even with that option turned off on Stream.me chat, anonymous viewers still are able to post on your program.

**Comment by xedoc** [comment]
Oh, I didn't mention that, will check it. Thank you!

**Comment by Draygera** [comment]
No problem. I'm trying to help as much as I possibly can. Still need to read through your code and figure out how things work.

**Comment by Draygera** [comment]
Did you check more into this?

**Comment by xedoc** [comment]
Yes, I don't remember details but there was some problem with implementation. Need to check it again :)
